notebooks/train_yolov7.ipynb

```diff
@@ -172,118 +178,79 @@
     "    vallabelsdir = create_directory('val', labelsdir)  # /workspace/dataset/labels/val\n",
     "    testlabelsdir = create_directory('test', labelsdir)  # /workspace/dataset/labels/test\n",
     "\n",
-    "    # Remove lost and found folder\n",
-    "    try:\n",
-    "        Path.rmdir(\"/workspace/lost+found\")\n",
-    "    except:\n",
-    "        print(\"Could Not Remove 'lost+found'\")\n",
+    "    # Connect to the Label Studio API and check the connection\n",
+    "    ls = teknoir_labelstudio_sdk.Client(\n",
+    "        extra_headers=teknoir_labelstudio_sdk.jwt_header('kubeflow.training@teknoir.ai'))\n",
+    "    print(ls.check_connection())\n",
     "\n",
-    "    bucket_name = f\"{namespace}.{domain}\"\n",
+    "    projects = ls.list_projects(title=project)\n",
+    "    p = projects[0]\n",
+    "    print(p.parsed_label_config)\n",
     "\n",
-    "    print(namespace)\n",
-    "    print(domain)\n",
-    "    print(bucket_name)\n",
+    "    yolo_annotations_file_zip = p.export_tasks(export_type=\"YOLO\", download_resources=True,\n",
+    "                                                     export_location=f'{workspace}/{project}.zip')\n",
     "\n",
-    "    storage_client = storage.Client()\n",
-    "    bucket = storage_client.get_bucket(bucket_name)\n",
+    "    tasks_json_file = p.export_tasks(export_type=\"JSON\", download_resources=True,\n",
+    "                                           export_location=f'{workspace}/{project}.json')\n",
     "\n",
-    "    my_prefix = f\"label-studio/projects/{project}/results/\"  # the name of the subfolder\n",
-    "    blobs = list(bucket.list_blobs(prefix=my_prefix, delimiter='/'))\n",
+    "    with open(tasks_json_file) as f:\n",
+    "        tasks = json.load(f)\n",
     "\n",
-    "    if len(blobs) == 0:\n",
-    "        print(\"NO RESULTS FOUND FOR {}\".format(project))\n",
-    "        # output = namedtuple('YOLOArgs', ['data_file', 'config_file', 'transfer_weights_file', 'hyp_file', 'names_file'])\n",
-    "        # return output(\"\", \"\", \"\", \"\", \"\")\n",
-    "        return\n",
+    "    if len(tasks) == 0:\n",
+    "        print(f'NO TASKS FOUND FOR {project}')\n",
+    "        exit(0)\n",
     "\n",
+    "    ind_list = list(range(len(tasks)))\n",
+    "    shuffle(ind_list)\n",
     "    test_frac = 1.0 - train_frac - validate_frac\n",
```

Comment:
do we need a test/validation split here? Unless we adjust model hyperparameters and fine tune those, we wouldnt need a final test set?

Reply:
Those arguments are parameterized and can be set arbitrarily for each training run.